doc/installationguide.md

```diff
@@ -41,6 +41,17 @@ These are the parameters that can be configured in the global section:
 }
 ```
 
+-   If you want to support a "mixed" mode with both **NGSI-v2** and **NGSI-LD** (experimental):
```

Comment:
I'd suggest to add an explanation of what mixed mode means. For instance:

> In mixed mode the NGSI version to use is chosen at group or device provisioning time, instead of using an static setting for that. The `ngsiVersion` field in the provisioning API is used to specify this, at both group at device level (the device level overriding the group setting).

It is just an example, it can be improved, of course.

Reply:
Fixed 52cb089